src/HtmlRenderer.php

```diff
@@ -85,4 +88,19 @@ private function createDirectory($suffix = "") {
         }
     }
 
+    /**
+     * @param $fileName
+     * @return string
+     */
+    private function createRelativePath($fileName)
+    {
+        $explodedFileName = explode("/", $fileName);
```

Comment:
Shouldn't we use DS as a separator here as well? I guess this hard coded '/' would cause problems on windows systems.

Reply:
You are right, I'll update it